patroni/postgresql/postmaster.py

```diff
@@ -114,7 +114,10 @@ def signal_stop(self, mode):
             logger.warning("Cannot stop server; single-user server is running (PID: {0})".format(self.pid))
             return False
         try:
-            self.send_signal(STOP_SIGNALS[mode])
+            if os.name != 'posix':
+                return self.pg_ctl_stop(mode, bin_dir)
```

Comment:
It would be better to move a call of `self.pg_ctl_kill` outside of try...catch block.